docs/src/components/ThemeExample.tsx

```diff
@@ -0,0 +1,25 @@
+import { Link, View } from '@aws-amplify/ui-react';
+import { ThemeAlert } from '@/components/ThemeAlert';
+import { GITHUB_REPO_FILE } from '@/data/links';
+import { DesignTokenIcon } from '@/components/Icons';
+
+export const ThemeExample = (props) => (
+  <>
+    ### Theme
```

Comment:
Asking out of my own ignorance around how our docs are complied but is markdown syntax supported within our `.tsx` files?

Reply:
I was going to ask this as well...

Reply:
Am curious what the linting error is? It may be a rule that doesn't make sense for us and could be turned off

Reply:
!! Good catch! Fixed!

Reply:
> Am curious what the linting error is? It may be a rule that doesn't make sense for us and could be turned off

Yea, I was thinking about that too. Then I decided to convert to tsx is because 1) this is the only .mdx file we have in the docs/components/, 2) we may convert all the .mdx files to .tsx in the future

Reply:
Still curious what is the actual rule/error? For context, linting libraries tend to be opinionated and some times those opinions do not align what our needs are

Reply:
I think it's a rule from https://github.com/aws-amplify/amplify-ui/pull/3369

Reply:
> Still curious what is the actual rule/error? For context, linting libraries tend to be opinionated and some times those opinions do not align what our needs are

We would have this error

> Expected an assignment or function call and instead saw an expression.eslint[no-unused-expressions](https://eslint.org/docs/rules/no-unused-expressions)

![image](https://user-images.githubusercontent.com/11983489/216465853-2bebf353-eac9-491b-a01a-0a8a79e069ee.png)

If you pull https://github.com/aws-amplify/amplify-ui/pull/3390 to local, I think you should be able to reproduce it.